<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pop_up</class>
 <widget class="QWidget" name="pop_up">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Un livre dont vous êtes le héro</string>
  </property>
  <widget class="QComboBox" name="comboBoxLivre">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="boutonNouvellePartie">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nouvelle Partie</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBienvenu">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>671</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>33</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Bienvenu loup solitaire</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxSave">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="boutonPartieSave">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Partie Sauvegardée</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boutonQuitter">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boutonPartieSupprimer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer partie</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
